SuperBuild/External_python-scipy.cmake

```diff
@@ -32,42 +32,48 @@ if(NOT Slicer_USE_SYSTEM_SciPy)
     set(Fortran_COMPILER_ID "Flang")
     find_package(Fortran REQUIRED)
 
+    set(BLAS_LIB ${LAPACK_DIR}/lib/blas.lib)
+    set(LAPACK_LIB ${LAPACK_DIR}/lib/lapack.lib)
+
     get_filename_component(flang_bin_path ${Fortran_Flang_EXECUTABLE} DIRECTORY)
 
     file(APPEND ${_env_script}
 "#------------------------------------------------------------------------------
 # Added by '${CMAKE_CURRENT_LIST_FILE}'
 
 set(ENV{LIB} \"\$ENV{LIB};${Fortran_Flang_IMPLICIT_LINK_DIRECTORIES}\")
-set(ENV{PATH} \"${flang_bin_path}\")
+set(ENV{PATH} \"\$ENV{PATH};${flang_bin_path}\")
 set(ENV{CC} \"${Fortran_Flang_CLANG_CL_EXECUTABLE}\")
 set(ENV{CXX} \"${Fortran_Flang_CLANG_CL_EXECUTABLE}\")
 set(ENV{FC} \"${Fortran_Flang_EXECUTABLE}\")
 
-set(ENV{BLAS} \"${LAPACK_DIR}/lib/blas.lib\")
-set(ENV{LAPACK} \"${LAPACK_DIR}/lib/lapack.lib\")
 set(ENV{ATLAS} \"\")
 ")
   else()
     # gfortran
     set(Fortran_COMPILER_ID "GNU")
     find_package(Fortran REQUIRED)
 
+    set(BLAS_LIB ${LAPACK_DIR}/lib/libblas.a)
+    set(LAPACK_LIB ${LAPACK_DIR}/lib/liblapack.a)
+
     get_filename_component(gnu_bin_path ${Fortran_GNU_EXECUTABLE} DIRECTORY)
 
     file(APPEND ${_env_script}
 "#------------------------------------------------------------------------------
 # Added by '${CMAKE_CURRENT_LIST_FILE}'
 set(ENV{PATH} \"${gnu_bin_path}:\$ENV{PATH}\")
 
-set(ENV{BLAS} \"${LAPACK_DIR}/lib/libblas.a\")
-set(ENV{LAPACK} \"${LAPACK_DIR}/lib/liblapack.a\")
 set(ENV{ATLAS} \"\")
 ")
   endif()
 
   # configure step
-  # NA
+  set(_config_script ${CMAKE_BINARY_DIR}/${proj}_config_step.cmake)
```

Comment:
Well done. Adding a comment explaining why this is done, and even indicating which numpy/scipy source files are involved would be great. That would allow the next person working on this to get up to speed faster.

Reply:
Also since we don't know for use `libs` is needed on Linux/macOS, let's add a comment indicated this.

For information, the structure of python-install on linux is the following:

```
$ find . -maxdepth `
```
.
./man
./bin
./lib
./include
./share

```

$ find . -maxdepth 2
.
./man
./man/man1
./bin
./bin/couchdb-load-design-doc
./bin/pyjwt
./bin/couchdb-dump
./bin/PythonSlicer
./bin/couchdb-load
./bin/SlicerPythonLauncherSettings.ini
./bin/python2.7
./bin/lsb_release
./bin/pip
./bin/nosetests-2.7
./bin/f2py
./bin/couchdb-replicate
./bin/chardetect
./bin/SlicerPythonLauncherSettingsToInstall.ini
./bin/easy_install-2.7
./bin/nosetests
./bin/pip2
./bin/pip2.7
./bin/libpython.py
./bin/PythonSlicerLauncherSettingsToInstall.ini
./bin/lsb_releaseLauncherSettings.ini
./bin/couchpy
./bin/SlicerPython
./bin/PythonSlicerLauncherSettings.ini
./bin/easy_install
./bin/python
./bin/lsb_releaseLauncherSettingsToInstall.ini
./bin/wheel
./bin/python2
./lib
./lib/pkgconfig
./lib/python2.7
./lib/libpython2.7.so
./include
./include/python2.7
./share
./share/man
./share/python2.7
```

Reply:
I changed libs to lib, still works on Windows and should also be in the path on Mac and Linux as well.